### DISCUSSION TITLE
1password Blocked

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#432] [Q&A]
Hi, this might sound like a very simple question, but how do I run an app that was blocked by defender application guard? I cant run 1password from winget or download from there website. Sorry very new to all this! I just ran your reccomended presets a couple days ago.

Thank you!

**Comment by HotCakeX** [comment]
Hi, can you please show me the block message you're seeing? because there's no defender application guard so i need to know which component is exactly blocking it.

**Reply by ghost** [reply]
It is a blue popup screen that says "Your orginazation used Windows Defender Application Control to block this app"

c:\users\name\Downloads\1PasswordSetup-latest.exe

**Reply by HotCakeX** [reply]
oh i see, you used the Downloads Defense measures category, it prevents executables/programs from running in the Downloads folder. The reason behind that is explained on the readme:  https://github.com/HotCakeX/Harden-Windows-Security?tab=readme-ov-file#downloads-defense-measures-

If you ever want to remove it, you can use the Unprotect tab from the user interface. If you trust the program you can copy/move it somewhere else from the default Downloads folder and then run it.

**Comment by ghost** [comment]
Does that help? I can take a picture too if need be.On Nov 30, 2024, at 9:55 AM, Violet Hansen ***@***.***> wrote:﻿
Hi, can you please show me the block message you're seeing? because there's no defender application guard so i need to know which component is exactly blocking it.

**Comment by ghost** [comment]
Oh Nice! Thank you!On Nov 30, 2024, at 10:41 AM, Violet Hansen ***@***.***> wrote:﻿
oh i see, you used the Downloads Defense measures category, it prevents executables/programs from running in the Downloads folder. The reason behind that is explained on the readme:  https://github.com/HotCakeX/Harden-Windows-Security?tab=readme-ov-file#downloads-defense-measures-
If you ever want to remove it, you can use the Unprotect tab from the user interface. If you trust the program you can copy/move it somewhere else from the default Downloads folder and then run it.

**Comment by ghost** [comment]
Also one last question! If I physically close my laptop it goes back to the bitlocker login. Is there away I can disable to that?Thanks again!On Nov 30, 2024, at 10:41 AM, Violet Hansen ***@***.***> wrote:﻿
oh i see, you used the Downloads Defense measures category, it prevents executables/programs from running in the Downloads folder. The reason behind that is explained on the readme:  https://github.com/HotCakeX/Harden-Windows-Security?tab=readme-ov-file#downloads-defense-measures-
If you ever want to remove it, you can use the Unprotect tab from the user interface. If you trust the program you can copy/move it somewhere else from the default Downloads folder and then run it.

**Reply by HotCakeX** [reply]
Yes, your laptop is probably configured to go to hibernate when the lid is closed, you can change that to sleep or if it supports modern standby use that instead of hibernate.

you can configure it in control panel, i highlighted the path and setting you need to click on

<img width="1280" alt="image" src="https://github.com/user-attachments/assets/2ec05499-318f-415f-906a-84252d1f4b36">

**Reply by HotCakeX** [reply]
If you want to disable BitLocker completely, you can use the same Unprotect tab again in the Harden Windows Security's GUI.

**Comment by ghost** [comment]
Thank you so much! It actually shows blank spots when I go the “what to do when the lid closes” page. Also for the life of me can’t find the bitlocker recovery code file!On Nov 30, 2024, at 10:58 AM, Violet Hansen ***@***.***> wrote:﻿
If you want to disable BitLocker completely, you can use the same Unprotect tab again in the Harden Windows Security's GUI.

**Reply by HotCakeX** [reply]
You're welcome, it shouldn't be empty, at least "shut down" and "do nothing" should be there

![image](https://github.com/user-attachments/assets/b3b7498a-6988-4ef0-bcfe-ac30159499fc)

The reason the "sleep" option might not be there is because the Microsoft security baselines enable a policy that disable the regular sleep mode (Power states S1-S3). [Briefly explained here](https://github.com/HotCakeX/Harden-Windows-Security?tab=readme-ov-file#bitlocker-settings). If the device supports modern standby mode (which is like phones, instant turn on) then you'd see sleep option in there too even if regular sleep is disabled.

<br>

You can get your BitLocker recovery keys from here by clicking/tapping on these areas

<img width="947" alt="image" src="https://github.com/user-attachments/assets/8580f66a-28ae-4379-b14f-29ea56035ac5">

<br>

You can disable BitLocker by first selecting the refresh button here to populate the list, then select a drive and click/tap on the decrypt button.

<img width="942" alt="image" src="https://github.com/user-attachments/assets/993de777-9c12-4133-8a9f-f0973d87e8fe">

<br>

**Comment by ghost** [comment]
Thank you! I thought it save a file somewhen to my c drive. I just wanted to make sure that it wasn’t easily accessible. Thank you for getting back to me so quick!On Dec 1, 2024, at 10:52 AM, Violet Hansen ***@***.***> wrote:﻿
You're welcome, it shouldn't be empty, at least "shut down" and "do nothing" should be there
image.png (view on web)
The reason the "sleep" option might not be there is because the Microsoft security baselines enable a policy that disable the regular sleep mode (Power states S1-S3). Briefly explained here. If the device supports modern standby mode (which is like phones, instant turn on) then you'd see sleep option in there too even if regular sleep is disabled.

You can get your BitLocker recovery keys from here by clicking/tapping on these areas
image.png (view on web)

You can disable BitLocker by first selecting the refresh button here to populate the list, then select a drive and click/tap on the decrypt button.
image.png (view on web)

**Reply by HotCakeX** [reply]
Anytime, happy to help!

**Comment by ghost** [comment]
One last question! Is there a clean OCD way to open an application while running downloads defense? Should I make a folder in the documents folder?On Dec 1, 2024, at 11:20 AM, Violet Hansen ***@***.***> wrote:﻿
Anytime, happy to help!

**Reply by HotCakeX** [reply]
After you install an application, it's no longer launched from the downloads folder, so the block is not recurring.
You can run applications in Windows Sandbox too.
Installing apps from Winget or MS Store aren't blocked by this app control policy.

**Comment by ghost** [comment]
So I actually first tried to download 1password from winstall and I till couldn’t open it. It was really weirdOn Dec 1, 2024, at 10:47 PM, Violet Hansen ***@***.***> wrote:﻿
After you install an application, it's no longer launched from the downloads folder, so the block is not recurring.
You can run applications in Windows Sandbox too.
Installing apps from Winget or MS Store aren't blocked by this app control policy.
